.github/stale.yml

```diff
@@ -6,7 +6,7 @@ daysUntilStale: 30
 # Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
 daysUntilClose: 7
 
-# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
```

Comment:
I would reword this a little. Just a suggestion:

```suggestion
# Only issues or pull requests with all of these labels are considered by StaleBot. Defaults to `[]` (disabled)
```

Reply:
Ok. I took the comment straight from the stalebot docs. But can reword if u prefer.

Reply:
Really? It was more just to remove the typo but I don't think it reads right anyway 🤔

Reply:
https://github.com/probot/stale 
I will take your changes, no problem!